test/include/RAJA_test-workgroup.hpp

```diff
@@ -360,8 +360,13 @@ using OpenMPTargetStoragePolicyList = SequentialStoragePolicyList;
 #if defined(RAJA_ENABLE_CUDA)
 using CudaExecPolicyList =
     camp::list<
+                #if defined(RAJA_TEST_EXHAUSTIVE)
+                // avoid compilation error:
+                // tpl/camp/include/camp/camp.hpp(104): error #456: excessive recursion at instantiation of class
                 RAJA::cuda_work<256>,
-                RAJA::cuda_work<1024>
+                #endif
```

Comment:
@MrBurmark @trws Is this ok? Removing this test case seems to make everything better. I could remove the 1024 test case instead.

Reply:
hmm, if we have to relegate one, I would keep 1024. But hopefully we can avoid this change.

Reply:
Yes, I kept 1024 in there a couple lines down. Github just has weird formatting, so it's difficult to see.

Reply:
That error in the comment makes me curious.  What would cause excessive recursion from this?

As to the counts, if this is testing blocks per SM then it probably needs a check to ensure the hardware can actually run that many and give an XFAIL or similar for the case where that's not possible.

Reply:
It's a set of WorkGroup tests which have 3*3*3*2 permutations of policies. Cutting it down to 2*3*3*2 seems to be fine for gcc/8.3.1 + cuda/10.1.243.

That particular count refers to the number of threads, although you're right it's a good idea to add a check for blocks per SM.

Reply:
Ok, if it's threads per block that's safe enough.  That's been supported since cc2.0.

Even if it's that many, camp shouldn't fail that way since the vast majority of the algorithms are non-recursive.  Mind filing some info on this, maybe a camp issue, so I can take a look at it?

Reply:
@trws Made a CAMP issue here https://github.com/LLNL/camp/issues/91. Thanks!